content/_index.en.md

```diff
@@ -4,8 +4,20 @@ title: Home
 
 # Geo-Distributed Systems
 
-Welcome! This is the landing page for [Rotational Labs'](https://rotational.io/) educational content on the implementation, deployment, and operation of globally distributed computing and storage systems!
+Welcome! This is the landing page for giti educational content on the implementation, deployment, and operation of globally distributed computing and storage systems!
```

Comment:
Looks like a typo has been injected here?

Reply:
Yes, let me correct it